src/content/protocols/pancakeswap-v2-bsc.md

```diff
@@ -0,0 +1,150 @@
+| ------------------- | -------- | ---------------------------------------------------------------------- |
+| Binance Smart Chain | L1 chain | All swaps & CAKE emission halt                                         |
+| CAKE token          | ERC-20   | Reward flow stops                                                      |
+| GnosisSafe multisig | Admin    | Parameter changes blocked, protocol fee's and CAKE emissions hijacked, |
+
+## Exit Window
+
+> Exit Window score: **Medium**
+
+PancakeSwap v2 exposes **three distinct paths** through which governance can change parameters that materially affect LPs or yield farmers and CAKE holders.
+Only two of those paths are enforced by the Timelock; the third is a direct call gated by the 3-of-6 multisig.
+
+| Path                                                                        | What can change                                                            | Who signs                      | Delay before execution        | Grace period            | User actions possible during delay                                             |
+| --------------------------------------------------------------------------- | -------------------------------------------------------------------------- | ------------------------------ | ----------------------------- | ----------------------- | ------------------------------------------------------------------------------ |
+| **Timelock → Factory**<br>`setFeeTo(...)`                                   | Redirects the 0.08 % protocol-fee stream of every pool to a new address    | 3-of-6 multisig via Timelock   | ≥ 6 h – ≤ 30 d (configurable) | 14 d, currently 8 hours | Remove liquidity, unwind positions                                             |
+| **Timelock → MasterChefV2**<br>`add / set / updateCakeRate / burnCake(...)` | Creates or reprioritises farms, changes CAKE emission splits and burn rate | 3-of-6 multisig via Timeloc√√k | ≥ 6 h – ≤ 30 d                | 14 d, currently 8 hours | Withdraw or migrate staked LPs                                                 |
+| **Multisig → MasterChefV1**<br>`dev(...)` (10 % mint share)                 | Changes the developer address that receives 10 % of every CAKE mint        | 3-of-6 multisig (direct)       | 0 s                           | —                       | LPs can still harvest/withdraw but inflation share can be redirected instantly |
+
+**Practical implications**
+
+- **User funds in pools** – `PancakePair` contracts are immutable; LPs can redeem liquidity at any time. Changing the protocol-fee address only affects fee distribution, not principal.
+- **Staked funds in MasterChef V1/V2** – Deposits and withdrawals remain permission-less, so users can exit as soon as they notice an unfavourable governance queue. The _effective_ exit window is therefore the Timelock delay (**≥ 6 h**).
+- **No freeze switch** – There is no guardian pause; the risk is dilution of CAKE rewards rather than loss or lock-up of tokens.
```

Comment:
I really appreciate how thorough this is but its doesn't exactly match our framework. It would be great if you could have distinct (short) paragraph for each of the 3 paths instead of table and bullet points.